test/LSM/pond_soil_lsm.jl

```diff
@@ -1,12 +1,5 @@
 using Test
-using UnPack
-using DiffEqCallbacks
-using OrdinaryDiffEq: ODEProblem, solve, Euler
```

Comment:
I like that we remove ODE.jl for tests, same reason, making it simpler is nice. And as you did, using a simple Euler manual can do the job.